REFERENCE.md

```diff
@@ -129,6 +148,7 @@ Data type: `Hash`
 A hash that holds all data for ferm::rule
 Default value: Empty Hash
 Allowed value: Any Hash
+Example: {'nat' => ['PREROUTING', 'POSTROUTING']}
```

Comment:
I think this is the example for the `preserve_chains_in_tables` parameter

Reply:
fixed it